src/Admin/FeaturedProgramAdmin.php

```diff
@@ -145,11 +147,21 @@ protected function configureFormFields(FormMapper $formMapper): void
       $file_options['help'] = '<img src="../'.$this->getFeaturedImageUrl($featured_project).'">';
       $id_value = $this->getSubject()->getProgram()->getId();
     }
-
+    $flavor_options = $this->parameter_bag->get('themes');
     $formMapper
       ->add('file', FileType::class, $file_options)
       ->add('program_id', TextType::class, ['mapped' => false, 'data' => $id_value])
-      ->add('flavor')
+      ->add('flavor', ChoiceFieldMaskType::class, [
```

Comment:
It would be really nice to generate the choices array automatically - else it will need work every time we add a new flavor. Something that should be possible with a few clicks in the admin interface in the future.

Reply:
![image](https://user-images.githubusercontent.com/40868718/81038208-7c0a0180-8ea5-11ea-83ae-29d74d63a059.png)

@leno12 already did it in his PR. You might wanna take a look at it.
![image](https://user-images.githubusercontent.com/40868718/81038218-8d530e00-8ea5-11ea-93e8-4ada4adeb342.png)

Reply:
Thanks, already done. Now I work on your other requests.